localization/english/event_localization/wc_central_kalimdor_events_l_english.yml

```diff
@@ -0,0 +1,250 @@
+cen_kal.1006.c.tt:0 "You order the goblins to work at a relaxed pace. While this means less income, it also means a safer enviroment with a lower risk of faliure."
+cen_kal.1006.d:0 "Halt all work on oil depot for now..."
+cen_kal.1006.d.tt:0 "You order the goblins to halt all work on the oil depot. While you'll be causing no further damage to the enviroment, the goblins will still need to be paid."
+cen_kal.1006.e:0 "Disable the depot permanently..."
+cen_kal.1006.e.tt:0 "You order the goblins to halt all work on the oil depot permanently and inform them that they should find employment opportunities elsewhere."
+
+cen_kal.1007.desc:0 "Your acting steward, [SCOPE.sC('steward').GetName] enters your council chamber and bows before you. They have brought you the yearly report on the oil depot managed by the blackwater goblins in [ROOT.Var('depot_location').Province.GetBaronyNameExplicitly]. They say:\n\n“My [ROOT.Char.GetTitleAsNameNoTooltip], I have prepared a report on the goblin oil depot, as you requested. It contains the data and information on the oil production, sales, income, expenses, and impact of the project for the year." You open the report and examine the data and information. \n\nHowever, your steward warns you that the reports received from the goblins cannot be fully trusted. They may exaggerate or falsify some of the numbers to suit their own agenda. They have probably also helped themselves to more than their fair share of the income from the project. Unfortunately, there is not much that can be done to verify their claims, as they are very secretive and protective of their operations, and you'll have to rely on their goodwill and honesty..."
+cen_kal.1007.a:0 "This is the only reason why I tolerate that [ROOT.Var('goblin_prospector').Char.GetFirstNameNoTooltip]..."
+
+cen_kal.1008.desc:0 "Your steward - [SCOPE.sC('steward').GetName] enters your council chamber for their weekly briefing and they begin to discuss the status of the goblin run oil depot located in [ROOT.Var('depot_location').Province.GetBaronyNameExplicitly]. You have invested a considerable deal of gold and resources in this project, and you want to make sure that its construction is proceeding smoothly and to schedule.\n\nYour steward notifies you that he has received a letter from [ROOT.Var('goblin_prospector').Char.GetFirstName] - the goblin leader in charge of the oil depot project in [ROOT.Var('depot_location').Province.GetBaronyNameExplicitly]. They mention how they have encountered some setbacks in the construction, and that they require more time and a further investment to pay off the workers. The goblin mentions how "bad weather" and the "hostile locals" are to blame for having fashioned a difficult workplace for the construction crew."
+cen_kal.1008.a:0 "The prospects are too great, we have little choice but to invest..."
+cen_kal.1008.b:0 "This venture is too costly, it's time we cut our losses..."
+
```

Comment:
You want to replace all the "marshal"s and "steward"s with scoped titles etc, so it's correct no matter what cultural names are involved.

Reply:
I believe the only way to do this is by having the scope on the council member himself. But that is not always possible as sometimes there is no councilor assigned so I pick a random courtier. Then this will throw an error. Albeit it is a low chance for someone to not have a councilor assigned it can happen. So what do you suggest I do?

Reply:
Well, I'd say your change is already what I requested :D